tests/Type/ArrayType/ArrayTypeTest.php

```diff
@@ -17,7 +18,10 @@ public function testStaticConstruct(): void
 		new ArrayType();
 	}
 
-	public function testContainsKeyIsNotStrict(): void
+	/**
+	 * @return mixed[][]|\Generator
+	 */
+	public function containsKeyIsNotStrictDataProvider(): Generator
 	{
 		$values = [
```

Comment:
Only immutable values should be reused at the top of data provider.

Plus arguably here it goes against this principle even more - most of the values in this array are not related to every case + if you imagine that there will end up being hundreds of cases in this DP, then:

1) you would always have to go again to the top to check what exactly is here
2) the array would have to change a lot + always would have to accommodate all cases making them needlessly dependent on each other